### DISCUSSION TITLE
React Server Components svgs being downloaded client side

### DISCUSSION THREAD

**Discussion by ricardo-fnd** [vercel/next.js#50627] [Help]
### Summary

Imported code on `server-only` files are being download in the browser.
This is happening with more svgs. Not all, but some, which is weird.

<img width="463" alt="Screenshot 2023-06-01 at 10 14 04" src="https://github.com/vercel/next.js/assets/52207061/bcc54fbf-cff1-4a84-9688-deca39789b99">
<img width="570" alt="Screenshot 2023-06-01 at 10 16 40" src="https://github.com/vercel/next.js/assets/52207061/e3fda660-6d27-43ff-8954-d29ce6bf1718">

### Additional information

If I coment the SVG import, the request on the browsers disappears.

**Comment by mcAnastasiou** [comment]
@ricardo-fnd  Any news on this?

**Reply by ricardo-fnd** [reply]
no sir...

**Reply by mcAnastasiou** [reply]
Ok. Thank you for the response. Should we create an issue instead?

**Reply by ricardo-fnd** [reply]
Yes, I need to test this again, as it was a long time ago. If its still happening I'l create the issue

**Reply by mcAnastasiou** [reply]
Thank you very much. I tested it and it is still happening...
